### DISCUSSION TITLE
User Request - Form Feedback

### DISCUSSION THREAD

**Discussion by JJediny** [GSA-TTS/GitHub-Administration#20] [General]
Please discuss the current [DRAFT/WIP User Request Form](https://github.com/GSA-TTS/GitHub-Administration/issues/new?assignees=tts-admins&labels=user%2Crequest&projects=&template=1-User.yml&title=Github+User+Request+-+email%40gsa.gov):
- Too Short?
- Too Long?
- What's Missing?
- ?

**Comment by JJediny** [comment]
Code of Conduct is a placeholder*

**Comment by akf** [comment]
> Government Furnished Equiptment (GFE) Computer?

Why this question? And are we asking whether the person has GFE (because they might need to use it to complete some process, presumably), or whether they're going to use it for this project? Because we know those are different answers for a lot of folks.
